backend/app/assets/javascripts/spree/backend/admin.js.erb

```diff
@@ -7,6 +7,7 @@
 //= require spree/backend/product_picker
 //= require spree/backend/option_value_picker
 //= require spree/backend/taxons
+//= require spree/backend/country_states
```

Comment:
If this country_states file isn't the same as the frontend's code for switching country/states (which I don't think it is) we should probably update the frontend to use it as well. I'd like to keep it DRY, which not sure how to best do keeping frontend/backend separate.  I don't think core should concern itself with this, but perhaps we should keep it in API?